### ISSUE TITLE
Typescript to Javascript cookbook hasn't be updated to reflect b.16 breaking changes

### ISSUE THREAD

**Issue by ericmartinezr** [angular/angular.io#1211]
https://angular.io/docs/js/latest/cookbook/ts-to-js.html

See the under **Importing and Exporting**

```javascript
var LocationStrategy =
  ng.router.LocationStrategy;
var HashLocationStrategy =
  ng.router.HashLocationStrategy;
```

It should be

```javascript
var LocationStrategy =
  ng.platform.common.LocationStrategy;
var HashLocationStrategy =
  ng.platform.common.HashLocationStrategy;
```

I don't know who wrote this cookbook so I can't ping him/her directly.

**Comment by filipesilva** [comment]
Included in #1226

**Comment by ericmartinezr** [comment]
@filipesilva thank you, I forgot about this issue, lol. I checked the website and now it's correct

```
var LocationStrategy =
  ng.common.LocationStrategy;
var HashLocationStrategy =
  ng.common.HashLocationStrategy;
```

Closing
